csm-1.5/CASMTRIAGE-7166-tapms-tenants-deploying/lib/setup-nexus.sh

```diff
@@ -0,0 +1 @@
+../../../csm-1.4/CASMCMS-9044-bos-cfs-scale/lib/setup-nexus.sh
```

Comment:
You probably don't want to symlink to my setup-nexus, because I have a lot of code in there for dealing with RPMs, and it doesn't look like your hotfix has any of that. I'd recommend symlinking to `csm-1.5/CASMINST-6896-gpg-keys/lib/setup-nexus.sh` instead